sky/engine/core/script/dart_init.cc

```diff
@@ -131,6 +131,52 @@ bool IsServiceIsolateURL(const char* url_name) {
          String(url_name) == DART_VM_SERVICE_ISOLATE_NAME;
 }
 
+#ifdef FLUTTER_PRODUCT_MODE
```

Comment:
This is tied to Dart's notion of "product mode", which disables features such as the observatory